package.json

```diff
@@ -0,0 +1,110 @@
+{
+  "name": "@mojaloop/thirdparty-api-adapter",
+  "version": "0.1.0",
```

Comment:
We should use the current versioning format here: `PI.SPRINT.INCREMENT`, which would be `10.4.0`

Reply:
Ok Fixed now.